src/processors/BaseProcessor.cpp

```diff
@@ -241,3 +241,36 @@ juce::Point<int> BaseProcessor::getPosition (Rectangle<int> parentBounds)
 {
     return (editorPosition * juce::Point { (float) parentBounds.getWidth(), (float) parentBounds.getHeight() }).toInt();
 }
+
+void BaseProcessor::makeBufferMono(AudioBuffer<float>& buffer)
+{
+    const auto nChannels = buffer.getNumChannels();
+    const auto numSamples = buffer.getNumSamples();
```

Comment:
Is it safe to assume this argument buffer will have the same number of samples as the other buffers that use this function, or should I include `numSamples` as another argument.

Reply:
Yeah, it's probably safe to expect that the buffers will have the same number of samples, but you might want to add a `jassert` just to be sure.

Reply:
Would the `jassert` be within the `makeBufferMono()` function? To have that wouldn't we need to include a `numSamples` argument for the function? Maybe that's what you were implying or I might be totally missing something.